docs/maintaining.md

```diff
@@ -2,15 +2,29 @@
 
 This document is for people working on VTEX as Toolbelt maintainers.
 
+**Table of Contents**
+
+- [Forcing a new version](#forcing-a-new-version)
+- [Deprecating a version](#deprecating-a-version)
+- [Changing the release message](#changing-the-release-message)
+- [Monitoring and debugging user errors](#monitoring-and-debugging-user-errors)
+
```

Comment:
Almost all changes on this document is formatting, except for this table of contents and the last section, which is new: Monitoring and debugging user errors